boards/calliope-mini/Makefile.include

```diff
@@ -1,5 +1,6 @@
 # for this board we support flashing via openocd or pyocd
 PROGRAMMER ?= openocd
+PROGRAMMERS_SUPPORTED ?= openocd jlink pyocd
```

Comment:
`jlink` and `openocd` are added in `include $(RIOTBOARD)/common/nrf51/Makefile.include`, why not move `pyocd` inclusion below?